src/components/RecipientCard.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <SmallPage class="recipient-card">
+        <Identicon :address="address"></Identicon>
+        <AddressDisplay :address="address"></AddressDisplay>
+        <div class="seperator"></div>
```

Comment:
Misspelled: `separator`
Semantically better would be to use a `hr` and style it accordingly.